readme.md

```diff
@@ -1,6 +1,9 @@
 Galacticraft - GTNH
 ===================
 
+# DEV NOTE: Micdoodlelib is integrated
+Do not include Micdoodlelib in your projects! It will break your projects!
```

Comment:
Maybe some explanation on why? `Lib X was completely integrated because of Y, make sure to not do Z`. Or at least make it a bit clear on what exactly projects (e.g. when having Galacticraft dev space or integrating with it) and how shouldn't you include it (e.g. don't put in minecraft's mods folder)